lang/en/scormremote.php

```diff
@@ -101,3 +101,14 @@
 $string['errorpage_sublimittitle'] = 'Subscription limit reached';
 $string['errorpage_sublimitmessage'] = 'Subscription limit has been reached. Contact your teacher to resolve this problem.';
 $string['privacy:metadata'] = 'The scorm remote module does not store any personal data.';
+$string['debugloglevel'] = 'Debugging log level';
+$string['debuglogleveldescription'] = 'One of DEBUG / INFO / WARN / ERROR (default) / NONE';
```

Comment:
Rather than listing the options I think this should have a sentence explaining what it actually does. eg When the log level is set it emits logs into the browser developer tools using console.log